src/plugins/ovp/components/medialive-channel-state-button/selector.js

```diff
@@ -0,0 +1,16 @@
+import get from 'lodash/get';
+import isGranted from '@triniti/cms/plugins/iam/selectors/isGranted';
+import getMediaLive from '@triniti/cms/plugins/ovp/selectors/getMediaLive';
+import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';
+import ChannelState from '@triniti/schemas/triniti/ovp.medialive/enums/ChannelState';
+
+/**
+ * @param {Object} state
+ * @param {Object} ownProps
+ *
+ * @returns {Object}
+ */
+export default (state, { node }) => ({
+  isPermissionGranted: isGranted(state, 'triniti:ovp.medialive:command:*'),
+  state: get(getMediaLive(state, NodeRef.fromNode(node)), 'state', ChannelState.UNKNOWN.getValue()),
```

Comment:
@efjacobson might want to call this channelState to remove confusion with react state, redux state and the state of california